outreach/gsoc/2024/idea-list.md

```diff
@@ -61,6 +61,7 @@ This idea aims to replace the current parser to get rid of a host of parser rela
 
 ### Description:
 
-Wayshot allows users to perform screen capture / streaming using CPU-copyback which leads to us being bottlenecked by the CPU throughput. This also includes image transformations on the CPU.
+Currently Wayshot performs screen capturing and streaming through the CPU causing bottlenecking and severe performance degradation.
 
-With the introduction of a DMA-BUF backend, we aim to expose a client-size API for directly capturing and performing all image-transformations at the GPU level. This enables high performance streaming over applications like OBS using our custom desktop portal backend.
+Through the introduction of a DMA-BUF backend and a client-side API for performing image capturing and transformations on the GPU. 
+This idea will enable high performance streaming over applications like OBS using our custom desktop portal backend.
```

Comment:
Since we're touching this line anyway, would you mind accomodating the project link too? 

https://github.com/waycrate/xdg-desktop-portal-luminous

Reply:
I will. Do you think this last sentence is necessary? I understand if you want to keep, I guess I am just asking for the sake of asking.

Reply:
So the latter is an optional, but the point of introducing the dmabuf backend is to improve our portal performance too. 

Not implementing the portal stuff is fine.

Reply:
I hope that makes it less ambiguous.

Reply:
Sorry, When I mean last sentence I mean this section
>This idea will enable high performance streaming over applications like OBS using our custom desktop portal backend.

Do you want this to be an example or a requirement of an outcome of the backend and api?

Reply:
It's an example outcome. Implementing it would be a huge plus.

Reply:
Understood